App.tsx

```diff
@@ -33,6 +34,15 @@ export default function App() {
               ),
             }}
             component={PlaygroundPage} />
+          <RootStack.Screen 
+            name="CreateTask"
+            options={{
+              title: "Recieve Help",
+              header: (headerProps) => (
+                <AppBar headerProps={headerProps} actionIcon={"send"} onActionClick={() => {} /*TODO*/}/>
+              ),
+            }}
+            component={CreateTask} />
         </RootStack.Navigator>
```

Comment:
Wrap the RootStack.Navigator with:
```typescript
        <KeyboardAvoidingView
            behavior={Platform.OS == "ios" ? "padding" : "height"}
            style={{flex: 1}}
        > ...</KeyboardAvoidingView>
```
This will make the bottom content in create-task vissable when the keyboard is up. Maybe check Android to if it looks fine.